apps/core/scenes/search/PlacePicker/PlacePickerRenderer.js

```diff
@@ -4,27 +4,32 @@ import * as React from 'react';
 import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
 
 import type { PlacePickerRendererQueryResponse } from './__generated__/PlacePickerRendererQuery.graphql';
-import PlacePickerContent from './PlacePickerContent';
+import PlacePickerRefetchContainer from './PlacePickerRefetchContainer';
+import { type Location } from '../SearchContext';
 
 type Props = {|
-  +type: string,
-  +defaultValue: ?string,
+  +defaultPlace: ?Location,
 |};
 
 export default class PlacePickerRenderer extends React.Component<Props> {
   renderInner = (data: PlacePickerRendererQueryResponse) => {
-    return <PlacePickerContent type={this.props.type} locations={data} />;
+    return <PlacePickerRefetchContainer locations={data} />;
+  };
+
+  setInputVariable = () => {
+    const term = this.props.defaultPlace?.locationId || '';
+    return { input: { term } };
   };
 
   render() {
     return (
       <QueryRenderer
         query={graphql`
           query PlacePickerRendererQuery($input: LocationsByTermInput!) {
-            ...PlacePickerContent_locations @arguments(input: $input)
+            ...PlacePickerRefetchContainer_locations @arguments(input: $input)
           }
         `}
-        variables={{ input: { term: this.props.defaultValue ?? '' } }}
+        variables={this.setInputVariable()}
```

Comment:
You are not setting anything in this function, you are however getting the variables. The name of the function is misleading.

Reply:
You are right 👍, thanks I didn't realize that, but definitely, this is about getting.

Reply:
done